docs/apis-hook.md

```diff
@@ -4,4 +4,149 @@ sidebar_position: 4
 
 # APIs hook
 
-This page will discuss - How to interact with the ReactPlay data using custom hook.
+In React Play, we work with APIs by using our custom-made react hooks. They are as follows:
+
+1. [useCacheResponse](#usecacheresponse)
+1. [useContributors](#usecontributors)
+1. [useFeaturedPlays](#usefeaturedplays)
+1. [useFetch](#usefetch)
+1. [useGetPlays](#usegetplays)
+1. [useGitHub](#usegithub)
+1. [useLikePlays](#uselikeplays)
+1. [useLocalStorage](#uselocalstorage)
+
+In this section, we will discuss each of them in detail.
+
+## useCacheResponse
+
+As the name suggests, this hook is used to create and maintain a cached version of the data.
+
+```JavaScript
```

Comment:
@supminn is this tag correct? The code part is not appearing with colors for keywords, name etc.

just check if it should be ```js instead.

Reply:
Yes, `jsx` worked well for this use-case.